plugins/DuskVerb/tools/tuner/ab_fdn_vs_accuratehall.py

```
@@ -40,8 +44,8 @@ def render(name, algo10, dst):
         try: os.remove(f)
         except OSError: pass
     args = [REND, "--program", name, *WET]
-    if algo10: args += ["--param", "Algorithm=1.0"]
-    r = subprocess.run(args, cwd=REPO, capture_output=True, text=True)
+    if algo10: args += ["--param", "Algorithm=10"]
```

Ruff (0.15.18)

[error] Multiple statements on one line (colon)

(E701)